loki/transformations/utilities.py

```diff
@@ -549,7 +557,7 @@ def get_loop_bounds(routine, dimension):
     :any:`Subroutine`.
 
     Parameters
-    ----------
```

Comment:
Rogue character removal?

Reply:
Fair enough ... thanks for marking this! Should be resolved.